datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -711,10 +724,15 @@ impl DisplayAs for ParquetExec {
                         )
                     })
                     .unwrap_or_default();
-
+                let dynamic_filter =
+                    format!("dynamic_filter: {:?}", self.dynamic_filters);
```

Comment:
I think it's best to only write this if available. + Would be good to implement `Display` for `DynamicFilterInfo` as well

Reply:
Sorry for that, this is for debug use, I'll refactor it